### ISSUE TITLE
Pageview Limits and Overages Doc Update

### ISSUE THREAD

**Issue by rachelwhitton** [pantheon-systems/documentation#3091]
Re: [Pageview Limits and Overages](https://pantheon.io/docs/preventing-plan-abuse/)

Priority (Low‚ Medium‚ High): Med-high

## Issue Description:
Users are asking about bandwidth usage in mbs, and we're taking the position that we don't offer that insight #3064 The next logical question in their mind is, well how about pageviews? They're looking for some kind of usage data on Pantheon to help them determine needs for external services 

ex scenario: they require an EV cert and need to stack CDNs, and need help gauging what plan to pick on that external CDN service

## Suggested Resolution 
Set expectations on whether or not users are exposed to pageviews. If so, how and when / under what circumstances? If not, can they request usage stats from support?

Maybe a new FAQ section titled "Can I see my pageview usage?" or something

@joshkoenig can you weight in? 

cc @tinefin @eabquina

**Comment by rachelwhitton** [comment]
Adding another question from slack by @stevector: Do we measure cached responses as a pageview?

**Comment by stevector** [comment]
My understanding is that, yes, cached responses count as pageviews because we want to keep pricing related metrics close to business value for the end owner of the website. And for that audience it may not be helpful to delineate between cached/uncached responses.

I don't know if caching needs it's own heading or just a sentence in the intro.

**Comment by rubelyn** [comment]
A customer is also asking if cached responses counts as pageviews - [#96734]. It was not mentioned in our doc. It is nice to have this info included.

**Comment by rachelwhitton** [comment]
Ping @joshkoenig - can you review this issue? 

TLDR; are users exposed to pageviews? if not, can they contact support to get pageview insights? does a cached response qualify as a pageview?

**Comment by joshkoenig** [comment]
Sorry missed this. 

The answer is yes. All pageviews are pageviews. 

Unless you are contacted by support, there's no need to worry. Regular chat and ticket support operators will not have access to this data.

Customer visibility is coming this year.

**Comment by alexfornuto** [comment]
Proposed resolution: FAQ info saying that:

 -  The Platform does not provide pageview data
 - CSE does not have pageview data
 - Cached pageviews are pageviews.
